src/storage/src/controller.rs

```diff
@@ -1456,7 +1457,10 @@ mod persist_read_handles {
                             let futs = FuturesUnordered::new();
                             for (_id, read) in read_handles.iter_mut() {
                                 if let Some(read) = read.as_mut() {
-                                    futs.push(read.maybe_heartbeat_reader());
+                                    // downgrade_since is idempotent, so downgrading to the minimum
+                                    // element is a no-op and can be used as a lease liveness
+                                    // heartbeat.
```

Comment:
using the min-element seems like something that should be in the downgrade_since docs. right now we only reference using `read.since()` to no-op heartbeat

Reply:
👍 will do

Reply:
done!